### PR TITLE
Multidomainsetup v2

### PR DESCRIPTION
**Pull request by leofuehringer**
It is possible to set a folder uid in the page TSconfig, which contains the categories, places, etc. Only this elements are shown in flexforms and TCA. It is helpful when a multi domain setup is used.

### CODE REVIEW

**Comment by leofuehringer** [comment]
Hi Oliver,
I am starting a new attempt to bring my ideas into your project.

Maybe you can take a look?
I still have one problem: For the flexforms I need markers in the /Configuration/FlexForms/flexforms_pi1.xml file.

However, I can't get the marker ###PAGE_TSCONFIG_STR### to be addressed via TSconfig.

Do you have a tip?
It must have the following form:
TCEFORM.tx_myext_table.myfield.PAGE_TSCONFIG_STR = %hello%

Have tried the following with no success:
TCEFORM.tt_content.pi_flexform.PAGE_TSCONFIG_STR=384

Thanks and have a nice weekend
Leo

File: Configuration/FlexForms/flexforms_pi1.xml [hunk 1]
```diff
@@ -350,6 +350,7 @@
 							<config>
 								<type>select</type>
 								<foreign_table>tx_seminars_event_types</foreign_table>
+				                <foreign_table_where>AND tx_seminars_event_types.pid LIKE ###PAGE_TSCONFIG_STR### ORDER BY title</foreign_table_where>
```
Comment (oliverklee) [inline Configuration/FlexForms/flexforms_pi1.xml:353]: If this is only a single UID, this shouldn't use a LIKE query, but instead a direct comparison. (Also, LIKE is relatively slow AFAIK.)

**Review by oliverklee — COMMENTED** [review]
I'm not quite happy with there always being an AND in the queries. Is there any way we could avoid that? Maybe with some custom PHP that gets called?

**Comment by oliverklee** [comment]
@leofuehringer Are you still planning to continue this PR, or can I close it?

**Comment by oliverklee** [comment]
Closing as there has been no response.

@leofuehringer If you want to continue with this, please feel free to reopen this PR or submit a new one!

**Pull request closed without merging**